api-schema/xcherry.yaml

```diff
@@ -685,15 +685,15 @@ components:
           items:
             type: string
         lockType:
```

Comment:
should we also rename this to "lockingType" to be consistent?

Reply:
good catch. Renamed to "lockType" (shorter).